Sources/LiCore/Networking/LiRestClient.swift

```diff
@@ -107,4 +95,16 @@ class LiRestClient {
         }
         return false
     }
+    fileprivate func logBeaconCall(_ response: (DataResponse<Any>)) {
+        if let path = response.request?.url?.path.contains("becon") {
+            if path {
```

Comment:
aren't you already checking for path in the above condition?

Reply:
Will update this later, based on our offline discussion.